### ISSUE TITLE
breakpad_linux: backtrace ids fail and no linux crash ids are written locally to the users upload.log file

### ISSUE THREAD

**Issue by keur** [brave/brave-browser#14326]
## Steps to Reproduce

   1. Open browser process from the CLI
   2. Navigate to the url `brave://inducebrowsercrashforrealz`
   3. Watch the logs

## Actual result:
```
--2021-02-22 12:36:38--  https://cr.brave.com
Resolving http://cr.brave.com (http://cr.brave.com)... 151.101.42.137, 2a04:4e42:a::649
Connecting to http://cr.brave.com (http://cr.brave.com)|151.101.42.137|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 36
Saving to: '/dev/fd/4'
     0K             
  Failed to get crash dump id. 
 Report Id:  ae310000-ab7a-e3 
```

The backtrace report id is truncated on stdout, and nothing is written to the file `~/.config/BraveSoftware/Brave-Browser-Development/Crash\ Reports/uploads.log`

## Expected result:

```
--2021-02-23 12:30:46--  https://cr.brave.com/
Loaded CA certificate '/etc/ssl/certs/ca-certificates.crt'
Resolving cr.brave.com (cr.brave.com)... 151.101.26.137, 2a04:4e42:a::649
Connecting to cr.brave.com (cr.brave.com)|151.101.26.137|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 36
Saving to: ‘/dev/fd/4’

/dev/fd/4                                       100%[====================================================================================================>]      36  --.-KB/s    in 0s      


2021-02-23 12:30:46 (416 KB/s) - ‘/dev/fd/4’ saved [36/36]

Crash dump id: 983b0000-ab7a-e305-0000-000000000000
[1]    112181 abort (core dumped)  ./src/out/Component/brave --enable-crash-reporter-for-testing 
```

Crash dump id is not truncated. The file `~/.config/BraveSoftware/Brave-Browser-Development/Crash\ Reports/uploads.log` has a new line with the ID `983b0000-ab7a-e305-0000-000000000000`

## Reproduces how often: 
Always

## Brave version (brave://version info)
All versions

## Version/Channel Information:
Affects all channels

**Comment by btlechowski** [comment]
Changed the milestone as the issue is fixed on 1.23.x and not on 1.22.x. I checked the commit history to confirm.

**Comment by btlechowski** [comment]
Verification passed on

Brave | 1.22.53 Chromium: 89.0.4389.72 (Official Build) beta (64-bit)
-- | --
Revision | 3f345f156bfd157bd1bea06310e55f3fb2490359-refs/branch-heads/4389@{#1393}
OS | Ubuntu 18.04 LTS

Verified test plan from the description

```
--2021-03-07 11:08:30--  https://cr.brave.com/
Resolving cr.brave.com (cr.brave.com)... 199.232.18.137
Connecting to cr.brave.com (cr.brave.com)|199.232.18.137|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 36
Saving to: ‘/dev/fd/4’

     0K                        
  Crash dump id:   1f1e0000-bef0-f505-0000-000000000000 
                          100%  764K=0s

2021-03-07 11:08:32 (764 KB/s) - ‘/dev/fd/4’ saved [36/36]

--2021-03-07 11:08:32--  https://cr.brave.com/
Resolving cr.brave.com (cr.brave.com)... 199.232.18.137
Connecting to cr.brave.com (cr.brave.com)|199.232.18.137|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 36
Saving to: ‘/dev/fd/4’

     0K                                
  Crash dump id:  201e0000-bef0-f505-0000-000000000000 
                   100%  565K=0s

2021-03-07 11:08:34 (565 KB/s) - ‘/dev/fd/4’ saved [36/36]
```

![image](https://user-images.githubusercontent.com/34715963/110236398-eceecf80-7f35-11eb-90c9-02c3619bdaaa.png)
